dataland-frontend/tests/e2e/specs/search-company-framework-data/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -34,36 +35,39 @@ describe("As a user, I expect the search functionality on the /companies page to
     cy.get("#framework-filter")
       .click()
       .get("div.p-multiselect-panel")
-      .find("li.p-highlight:contains('EU Taxonomy for financial companies')")
+      .find(`li.p-highlight:contains(${humanizeString(DataTypeEnum.EutaxonomyFinancials)})`)
       .click();
     verifySearchResultTable();
     cy.url()
       .should(
         "eq",
         getBaseUrl() +
           "/companies?" +
-          `framework=${DataTypeEnum.P2p}` +
-          `&framework=${DataTypeEnum.Sme}` +
+          `framework=${DataTypeEnum.EutaxonomyNonFinancials}` +
+          `&framework=${DataTypeEnum.Lksg}` +
+          `&framework=${DataTypeEnum.P2p}` +
           `&framework=${DataTypeEnum.Sfdr}` +
-          `&framework=${DataTypeEnum.Lksg}`,
+          `&framework=${DataTypeEnum.Sme}`,
       )
       .get("div.p-multiselect-panel")
-      .find("li.p-multiselect-item:contains('EU Taxonomy for financial companies')")
+      .find(`li.p-multiselect-item:contains(${humanizeString(DataTypeEnum.EutaxonomyFinancials)})`)
       .click();
     verifySearchResultTable();
     cy.url()
       .should("eq", getBaseUrl() + "/companies")
       .get("div.p-multiselect-panel")
-      .find("li.p-highlight:contains('SFDR')")
+      .find(`li.p-highlight:contains(${humanizeString(DataTypeEnum.Sfdr)})`)
```

Comment:
Why were these changes necessary, is `humanizeString(framework)` not working as expected anymore?

Reply:
These changes here are not necessary.  They are just better code.
Instead of hardcoding "SFDR" or "EU Taxonomy for financial companies", you can simply use humanizeString and the DataType Enum to retrieve the desired string.

And it is better because it automatically adjusts to changes in the DataType Enum.             Call me if you want a more detailed explanation.